qt/python/mantidqt/widgets/tableworkspacedisplay/test_helpers/mock_plotlib.py

```diff
@@ -0,0 +1,30 @@
+from mock import Mock
```

Comment:
Nice use of mocking out the plotting functionality. We're missing the licence header.

Can we have some module or class documentation to explain its purpose?